templates/amq-broker-71-custom.yaml

```diff
@@ -128,6 +128,8 @@ objects:
             value: ${AMQ_GLOBAL_MAX_SIZE}
           - name: AMQ_ALLOW_ANONYMOUS
             value: ${AMQ_ALLOW_ANONYMOUS}
+          - name: AMQ_DATA_DIR
+            value: ${AMQ_DATA_DIR}
```

Comment:
If we add the env, we will also need to add a parameter with a default.  However, I believe it best to not include it in templates that don't use statefulsets.  As long as it isn't listed as an env, the data dir parameter won't be included, and the default location will be used.

Reply:
Creating a PR, to modify the above.